src/test/js/runtests.js

```diff
@@ -56,8 +56,8 @@ var logger = {
 }());
 
 function assertEqual(expected, actual) {
-  expected = JSON.stringify(expected);
-  actual = JSON.stringify(actual);
+  var expected = JSON.stringify(expected);
```

Comment:
These `var` are redundant. Overwriting the arguments was intended (if a bit clever).